pkgs/development/libraries/qt-6/srcs.nix

```diff
@@ -1,318 +1,318 @@
 # DO NOT EDIT! This file is generated automatically.
-# Command: /home/k900/gh/NixOS/nixpkgs/./maintainers/scripts/fetch-kde-qt.sh pkgs/development/libraries/qt-6
+# Command: /home/k900/nixpkgs/./maintainers/scripts/fetch-kde-qt.sh pkgs/development/libraries/qt-6
```

Comment:
Shouldn't we strip the absolute path of your home directory?

Reply:
Applied in https://github.com/NixOS/nixpkgs/pull/347268